jenkins/scripts/select-compiler.sh

```diff
@@ -73,8 +73,24 @@ elif [ "$SELECT_ARCH" = "S390X" ]; then
   fi
 
 elif [ "$SELECT_ARCH" = "AIXPPC" ]; then
-  # get node version
-  echo "Setting compiler for Node version $NODEJS_MAJOR_VERSION on AIX"
+  case $NODE_NAME in
+    *aix72* )
+      echo "Setting compiler for Node version $NODEJS_MAJOR_VERSION on AIX72"
+
+      if [ "$NODEJS_MAJOR_VERSION" -gt "9" ]; then
+        export LIBPATH=/opt/gcc-6.3/lib/gcc/powerpc-ibm-aix7.2.0.0/6.3.0/pthread/ppc64:/opt/gcc-6.3/lib
+        export PATH="/opt/ccache-3.7.4/libexec:/opt/gcc-6.3/bin:$PATH"
+        export CC="`which gcc`" CXX="`which g++`" CXX_host="`which g++`"
+        echo "Compiler set to 6.3"
+	return
+      else
+        echo "Compiler defaulting to 4.8"
```

Comment:
This isn't correct on AIX 7.2? The compiler installed via yum from the AIX Toolbox on AIX 7.2 is gcc 8.

Reply:
fixed